config/utilities/constants.py

```diff
@@ -0,0 +1,330 @@
+# Copyright 2024 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+CONTAINERS = [{'name': 'Sarus',       'cmd': 'sarus'},
+              {'name': 'Apptainer',   'cmd': 'apptainer'},
+              {'name': 'Docker',      'cmd': 'docker'},
+              {'name': 'Singularity', 'cmd': 'singularity'},
+              {'name': 'Shifter',     'cmd': 'shifter'}]
+
+
+class bcolors:
+    HEADER = '\033[95m'
+    BLUE = '\033[94m'
+    CYAN = '\033[96m'
+    GREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+RFM_DOCUMENTATION = {'modules':              'https://reframe-hpc.readthedocs.io/en/stable/config_reference.html#config.systems.modules',
```

Comment:
instead of repeating `https://...`, can you use a variable ?
something like:
```
url = 'https://reframe-hpc.readthedocs.io/en/stable/'
...
'modules' = f'{url}/config_reference.html#config.systems.modules',
```